src/ts/waterfall/details-overlay/svg-details-overlay-manager.ts

```diff
@@ -46,11 +48,14 @@ export function closeOverlay(index: number, overlayHolder: SVGGElement,
 export function openOverlay(index: number, y: number, accordionHeight: number, entry: WaterfallEntry,
                             overlayHolder: SVGGElement, barEls: SVGGElement[]) {
 
-  if (openOverlays.filter((o) => o.index === index).length > 0) {
+  const overlayHolderId = overlayHolder.id;
+  if (openOverlays[overlayHolderId] && openOverlays[overlayHolderId].filter((o) => o.index === index).length > 0) {
```

Comment:
Maybe this is more readable as two if statements, and with some(). Something like:
```
if (!openOverlays[overlayHolderId]) {
      openOverlays[overlayHolderId] = [];
}
if (openOverlays[overlayHolderId].some((o) => o.index === index)) {
      return;
}
```